docs/edgeql/functions/converters.rst

```diff
@@ -8,6 +8,216 @@ These functions convert between different scalar types. When a
+    value can be parsed from the input :eql:type:`str` *s* with an
+    optional *fmt* argument or it can be given in terms of it's
+    component parts: *hour*, *min*, *sec*.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT to_naive_time('15:01:22.306916');
+        {<naive_time>'15:01:22.306916'}
+        db> SELECT to_naive_time('03:01:22pm', 'HH:MI:SSam');
+        {<naive_time>'15:01:22'}
+        db> SELECT to_naive_time(15, 1, 22.306916);
+        {<naive_time>'15:01:22.306916'}
+
+
+.. eql:function:: std::to_timedelta( \
+                    NAMED ONLY years: int64=0, \
+                    NAMED ONLY months: int64=0, \
+                    NAMED ONLY weeks: int64=0, \
+                    NAMED ONLY days: int64=0, \
+                    NAMED ONLY hours: int64=0, \
+                    NAMED ONLY mins: int64=0, \
+                    NAMED ONLY secs: float64=0 \
+                  ) -> timedelta
+
+    :index: timedelta
+
+    Create a :eql:type:`timedelta` value.
+
+    This function uses ``NAMED ONLY`` arguments  to create a
+    :eql:type:`timedelta` value. The available timedelta fields are:
+    *years*, *months*, *weeks*, *days*, *hours*, *mins*, *secs*.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT to_timedelta(hours := 1,
+        ...                     mins := 20,
+        ...                     secs := 45);
+        {<timedelta>'1:20:45'}
+        db> SELECT to_timedelta(secs := 4845);
+        {<timedelta>'1:20:45'}
+
+
+.. eql:function:: std::to_decimal(s: str, fmt: OPTIONAL str={}) -> decimal
+
+    :index: parse decimal
+
+    Create a :eql:type:`decimal` value.
+
+    Parse a :eql:type:`decimal` from the input *s* and optional format
+    specification *fmt*.
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT to_decimal('-000,012,345', 'S099,999,999,999');
+        {-12345n}
+        db> SELECT to_decimal('-012.345');
+        {-12.345n}
+        db> SELECT to_decimal('31st', '999th');
+        {31n}
+
+
+.. eql:function:: std::to_int16(s: str, fmt: OPTIONAL str={}) -> int16
+
+    :index: parse int16
+
+    Create a :eql:type:`int16` value.
+
+    Parse a :eql:type:`int16` from the input *s* and optional format
+    specification *fmt*.
+
+
+.. eql:function:: std::to_int32(s: str, fmt: OPTIONAL str={}) -> int32
+
+    :index: parse int32
+
+    Create a :eql:type:`int32` value.
+
+    Parse a :eql:type:`int32` from the input *s* and optional format
+    specification *fmt*.
+
+
+.. eql:function:: std::to_int64(s: str, fmt: OPTIONAL str={}) -> int64
+
+    :index: parse int64
+
+    Create a :eql:type:`int64` value.
+
+    Parse a :eql:type:`int64` from the input *s* and optional format
+    specification *fmt*.
+
+
+.. eql:function:: std::to_float32(s: str, fmt: OPTIONAL str={}) -> float32
+
+    :index: parse float32
+
+    Create a :eql:type:`float32` value.
+
+    Parse a :eql:type:`float32` from the input *s* and optional format
+    specification *fmt*.
```

Comment:
Can we copy/paste Postgres format specs to our docs?  See `tx_start.rst` on how to deal with copyright/licensing.

Reply:
Added format specs.